mods/pvp/killheal/init.lua

```diff
@@ -0,0 +1,8 @@
+local heal_val = 4 -- two hearts
+minetest.register_on_dieplayer(function(player, reason)
+    if reason.type == "punch" and minetest.is_player(reason.object) then
+        local killer = reason.object
+        if killer.get_player_name() == player.get_player_name() then return end
```

Comment:
Incorrect; using `.` causes a crash. This fixes it.
```suggestion
        if killer:get_player_name() == player:get_player_name() then return end
```